main.py

```diff
@@ -46,11 +46,11 @@
     default="shield-recipe-client/*, normandy/*, main/url-classifier-skip-urls",
 )
 
-# XXX What about instead just say that `bad == status.endswith('_error')`??
 GOOD_STATUSES = config("GOOD_STATUSES", cast=Csv(), default="success, up_to_date")
 
-# XXX What about 'backoff' ??
-NEUTRAL_STATUSES = config("NEUTRAL_STATUSES", cast=Csv(), default="pref_disabled")
+NEUTRAL_STATUSES = config(
+    "NEUTRAL_STATUSES", cast=Csv(), default="backoff, pref_disabled"
+)
```

Comment:
Is this needed anymore? If we have `GOOD_STATUSES` and `is_bad = status.endswith('_error')` then what's thee use of `NEUTRAL_STATUSES`? Because `pref_disabled`, for example, is naturally "neutral" even without this list. It's not in `GOOD_STATUSES` and it doesn't end with `_error`.

Feels like we need to pick either GOOD+NEUTRAL or GOOD+BAD.